File: javascript solutions/multiples3and5.js
var sum_multiples = function(limit) {
  var sum = 0;
  var counter = 0;
  for (var i = 0; i < limit; i++) {
    if (i % 5 == 0 || i % 3 == 0) {
      sum += i
    }
  }
  return sum;
}


console.log(sum_multiples(10))

File: ruby solutions/multiples3and5.rb

def sum_multiples(limit)
  sum = 0
  counter = 0

  until (counter == limit) do
    sum += counter if (counter % 5 == 0 || counter % 3 == 0)
    counter += 1
  end
  sum

end

 p sum_multiples(10) == 23
